apps/modernization-ui/src/apps/page-builder/components/AlertBase/AlertBase.tsx

```diff
@@ -0,0 +1,29 @@
+import './AlertBase.scss';
+import { Icon } from '@trussworks/react-uswds';
+import React from 'react';
+
+export type AlertBannerProps = {
+    type?: string;
+    children?: any;
+    message?: string;
+    header?: string;
+};
+
+export const AlertBase = ({ type, message, header }: AlertBannerProps) => {
+    return (
+        <div className={`usa-alert_body ${type}`}>
+            <div className="msg-container">
+                <span className="alert-left-cont">
+                    {type === 'success' && <Icon.CheckCircle size={3} />}
+                    {type === 'warning' && <Icon.Warning size={4} />}
+                    {type === 'prompt' && <Icon.Info size={3} />}
+                    {type === 'error' && <Icon.Error size={5} />}
+                </span>
+                <span>
+                    {header && <h2 className="header"> {header}</h2>}
```

Comment:
Potential Issue: I have heard from Adam that using `heading` tags can be a problem for accessibility and the general rule is to not use them.

Reply:
Removed this file